Code/Core/FileIO/IOStream.h

```diff
@@ -60,6 +57,35 @@ class IOStream
     // manage memory-mapped aligned data
     void AlignRead( size_t alignment );
     void AlignWrite( size_t alignment );
+
+    inline uint64_t WriteStringRaw( const AString& s ) { return WriteBuffer( s.begin(), s.GetLength() ); }
+
+   // Write several strings into IOStream.
+   //------------------------------------------------------------------------------
+   template < typename T0 = AString, typename... T >
+   bool WriteV( const T0& part0, const T&... parts )
+   {
+       bool rc = true;
+       rc = rc && Write( part0 );
+       if constexpr( sizeof...( parts ) > 0 ) {
+           rc = rc && WriteV( parts... );
+       }
+       return rc;
+   }
+
+   // Write several raw strings into IOStream (raw - means no length information will be stored in the stream)
+   //------------------------------------------------------------------------------
+   template < typename T0 = AString, typename... T >
+   uint64_t WriteRawV( const T0& part0, const T&... parts )
+   {
+       uint64_t rc = 0;
+       rc += WriteStringRaw( part0 );
```

Comment:
The function takes a template param for the type, but since it calls WriteStringRaw, it could only ever be used with an AString, so the template param seems unnecessary.

Reply:
A template (more specifically, a variadic template) was only used here to create functions that take variable number of arguments (see my comment above). As you correctly noted, it could only be used with AString.